invenio.cfg

```diff
@@ -189,21 +189,15 @@ CELERY_BEAT_SCHEDULE = {
         "task": "invenio_swh.tasks.cleanup_depositions",
         "schedule": crontab(minute=0, hour=6), # Every day at 06:00 UTC
     },
-    'hard-file-checks': {
-        'task': 'invenio_files_rest.tasks.schedule_checksum_verification',
-        'schedule': timedelta(hours=1),
-        'kwargs': {
-            'batch_interval': {'hours': 1},
-            # Manually check and calculate checksums of files biannually
-            'frequency': {'days': 180},
-            # Split batches based on total files size
-            'max_size': 0,
-            # Query taking into account only files with URI prefixes defined by
-            # the FILES_REST_CHECKSUM_VERIFICATION_URI_PREFIXES config variable
-            'files_query':
-                'invenio_app_rdm.utils.files.checksum_verification_files_query',
-            # Actual checksum calculation, instead of relying on a EOS query
-            'checksum_kwargs': {'use_default_impl': True},
+    # Overriding existing file-checks
+    "file-checks": {
+        "task": "invenio_files_rest.tasks.schedule_checksum_verification",
+        "schedule": timedelta(hours=1),
+        "kwargs": {
+            "batch_interval": {"hours": 1},
+            "frequency": {"days": 180},
+            "max_count": 0,
```

Comment:
max_count or max_size ?

Reply:
https://github.com/inveniosoftware/invenio-app-rdm/blob/ca315e52f5a09d5f85c01217c8921f293ef8b63c/invenio_app_rdm/config.py#L435-L446

max_count since we don't take size into account for these I suppose.